From 4f9b820244bfb52e58a11871986e7159e636d57e Mon Sep 17 00:00:00 2001
From: Thomas Preud'homme <thomas.preudhomme@celest.fr>
Date: Wed, 11 Aug 2010 10:45:08 +0200
Subject: Fix typo in manpage and source

Two function declarations suffer from a mispell: requred instead of required.
As a consequences, these typo are also found in the manpages generated by c2man.

Origin: vendor
Forworded: https://sourceforge.net/tracker/?func=detail&aid=3084926&group_id=37192&atid=419516
Last-Update: 2010-12-23
Applied-Upstream: http://vformat.cvs.sourceforge.net/viewvc/vformat/src/vformat/vf_iface.h?revision=1.41&view=markup
---
 doc/vf_get_prop_name_string.3  |    2 +-
 doc/vf_get_prop_value_string.3 |    2 +-
 vformat/vf_iface.h             |    4 ++--
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/doc/vf_get_prop_name_string.3 b/doc/vf_get_prop_name_string.3
index 7d1cfee..aa3bf3b 100644
--- a/doc/vf_get_prop_name_string.3
+++ b/doc/vf_get_prop_name_string.3
@@ -23,7 +23,7 @@ extern char *vf_get_prop_name_string
 Property to locate string from.
 .TP
 .B "uint32_t n_string"
-Index to string requred.
+Index to string required.
 .SH "DESCRIPTION"
 Get n'th name string.  For example a property may be defined as:
 
diff --git a/doc/vf_get_prop_value_string.3 b/doc/vf_get_prop_value_string.3
index 4fc9a81..afc6f66 100644
--- a/doc/vf_get_prop_value_string.3
+++ b/doc/vf_get_prop_value_string.3
@@ -23,7 +23,7 @@ extern char *vf_get_prop_value_string
 Property to locate string from.
 .TP
 .B "uint32_t n_string"
-Index to string requred.
+Index to string required.
 .SH "DESCRIPTION"
 Obtain string pointer value from VF_PROP_T.  For example a property
 may be defined as:
diff --git a/vformat/vf_iface.h b/vformat/vf_iface.h
index ab5e8d9..12c5f22 100644
--- a/vformat/vf_iface.h
+++ b/vformat/vf_iface.h
@@ -906,7 +906,7 @@ extern VFORMATDECLSPEC bool_t vf_set_prop_value(
 
 extern VFORMATDECLSPEC char *vf_get_prop_value_string(
     VF_PROP_T *p_prop,              /* Property to locate string from */
-    uint32_t n_string               /* Index to string requred */
+    uint32_t n_string               /* Index to string required */
     );
 
 
@@ -931,7 +931,7 @@ extern VFORMATDECLSPEC char *vf_get_prop_value_string(
 
 extern VFORMATDECLSPEC char *vf_get_prop_name_string(
     VF_PROP_T *p_prop,              /* Property to locate string from */
-    uint32_t n_string               /* Index to string requred */
+    uint32_t n_string               /* Index to string required */
     );
 
 
-- 
1.7.6.3

